Tests/CryptoTests/Signatures/ECDSA/ECDSASignatureTests.swift

```diff
@@ -78,37 +79,87 @@ class SignatureTests: XCTestCase {
                 bundleType: self,
                 jsonName: "ecdsa_secp256r1_sha512_test",
                 testFunction: { (group: ECDSATestGroup) in
-                    try orFail { try testGroup(group: group, curve: P256.Signing.self, hashFunction: SHA512.self) }
+                    try orFail { try testGroup(group: group, curve: P256.Signing.self, hashFunction: SHA512.self, deserializeSignature: P256.Signing.ECDSASignature.init(derRepresentation:)) }
                 })
         }
     }
     
-    func testWycheProofP384() throws {
+    func testWycheproofP384DER() throws {
         try orFail {
             try wycheproofTest(
                 bundleType: self,
                 jsonName: "ecdsa_secp384r1_sha384_test",
                 testFunction: { (group: ECDSATestGroup) in
-                    try orFail { try testGroup(group: group, curve: P384.Signing.self, hashFunction: SHA384.self) }
+                    try orFail { try testGroup(group: group, curve: P384.Signing.self, hashFunction: SHA384.self, deserializeSignature: P384.Signing.ECDSASignature.init(derRepresentation:)) }
                 })
         }
         try orFail {
             try wycheproofTest(
                 bundleType: self,
                 jsonName: "ecdsa_secp384r1_sha512_test",
                 testFunction: { (group: ECDSATestGroup) in
-                    try orFail { try testGroup(group: group, curve: P384.Signing.self, hashFunction: SHA512.self) }
+                    try orFail { try testGroup(group: group, curve: P384.Signing.self, hashFunction: SHA512.self, deserializeSignature: P384.Signing.ECDSASignature.init(derRepresentation:)) }
                 })
         }
     }
     
-    func testWycheProofP521() throws {
+    func testWycheproofP521DER() throws {
         try orFail {
             try wycheproofTest(
                 bundleType: self,
                 jsonName: "ecdsa_secp521r1_sha512_test",
                 testFunction: { (group: ECDSATestGroup) in
-                    try orFail { try testGroup(group: group, curve: P521.Signing.self, hashFunction: SHA512.self) }
+                    try orFail { try testGroup(group: group, curve: P521.Signing.self, hashFunction: SHA512.self, deserializeSignature: P521.Signing.ECDSASignature.init(derRepresentation:)) }
+                })
+        }
+    }
+    
+    func testWycheproofP256P1363() throws {
```

Comment:
@Lukasa let me know if you want these test function to have suffix `RawRepresentation` instead of `P1363`, since the initialiser being used is `init:rawRepresentation`. I chose `P1363` so to match the Wycheproof test vector names...

Reply:
We can leave the name as-is, I think it's fine.